parser/src/parser/grammar.lalrpop

```diff
@@ -240,14 +301,14 @@ IntegrityConstraints: Vec<IntegrityStmt> = {
     "integrity_constraints" ":" <integrity_stmts: IntegrityStmt+> =>? {
         // check if at least one integrity constraint is defined
         let integrity_constraints_exist = integrity_stmts.iter().any(|stmt| match stmt {
-            IntegrityStmt::Constraint(_) => true,
+            IntegrityStmt::Constraint(_) | IntegrityStmt::EvaluatorFunctionCall(_) => true,
             _ => false,
         });
         if !integrity_constraints_exist {
             return Err(ParseError::User {
                 error: Error::ParseError(
                     MissingIntegrityConstraint(
-                        "Declaration of at least one integrity constraint is required".to_string()
+                        "Declaration of at least one integrity constraint or evaluator function call is required".to_string()
```

Comment:
This looks like the same error message as above. It would be good to extract this so it's not duplicated. This could be improved as part of #188 though